.github/workflows/build_cmake.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           }
         - {
             name: "Ubuntu Build", artifact: "Linux.tar.xz",
-            os: ubuntu-20.04,
+            os: ubuntu-latest,
```

Comment:
> Update github hosted runners from ubuntu-20.04 to ubuntu-22.04

`ubuntu-latest` is currently 24.04, not 22.04. Some time next year, it will be updated to 26.04, which might break grpc-device builds.

Last time, @reckenro considered using `ubuntu-latest` and chose to continue using a specific version: https://github.com/ni/grpc-device/pull/901